scapy/contrib/gtp_v2.py

```diff
@@ -444,6 +446,16 @@ class IE_ULI(gtp.IE_Base):
     ]
 
 
+class IE_ULI_Timestamp(gtp.IE_Base):
+    name = "IE ULI Timestamp"
+    fields_desc = [
+        ByteEnumField("ietype", 170, IEType),
+        ShortField("length", None),
+        BitField("CR_flag", 0, 4),
+        BitField("instance", 0, 4),
+        BitField("Timestamp", 0, 32)]
```

Comment:
Please use IntField and only use lower case for the name.

Reply:
Done, changed timestamp to lowercase. Did not change other fields to keep the same formatting between the IEs